hack/deploy-karmada.sh

```diff
@@ -12,6 +12,11 @@ mkdir -p "${CERT_DIR}" &>/dev/null ||  mkdir -p "${CERT_DIR}"
 rm -f "${CERT_DIR}/*" &>/dev/null ||  rm -f "${CERT_DIR}/*"
 KARMADA_APISERVER_SECURE_PORT=${KARMADA_APISERVER_SECURE_PORT:-5443}
 
+# Define karmada control plane image registry and version
+KARMADA_REGISTRY=${KARMADA_REGISTRY:-"swr.ap-southeast-1.myhuaweicloud.com/karmada"}
+KARMADA_VERSION=${KARMADA_VERSION:-"latest"}
+KUBE_REGISTRY=${KUBE_REGISTRY:-"k8s.gcr.io"}
```

Comment:
I like the variate `KUBE_REGISTRY` very much that reduces my steps of deployment~

cc @RainbowMango